test/test_cli_integration.py

```diff
@@ -136,28 +142,32 @@ def generate_policy_string(self):
         variable_3 = 'simple/special @#$%^&*(){{}}[]._+/{id}'.format(id=uuid.uuid4().hex)
 
         policy = \
-"""
```

Comment:
IMO this change should be reverted even though the rendering of the policy will not error. As we previously spoke bout, `"""` is a sensitive formatter so I believe this policy would be rendered weird which is not necessary for the integrity of the test